sync_qubes-os.org_repo.sh

```diff
@@ -1,15 +1,25 @@
-#!/bin/sh
+#!/bin/bash --
+set -eu
 
-pushd `dirname $0`
+case $0 in
+(/*) dir=${0%/*}/;;
+(*/*) dir=./${0%/*};;
+(*) dir=.;;
+esac
```

Comment:
Why open-coding `dirname` instead of using existing thing?

Reply:
Mostly because `dirname` has some awkward corner cases.  For example, the naive way of using it fails if the directory ends with a newline.

Reply:
I'd prefer to not introduce perl-readable fragment just to handle corner cases that are not going to happen. The script is in qubes-src/linux-yum, it definitely does not ends with a newline. This code is just to allow calling the script being in another dir (like qubes-builder).